src/py42/services/cases.py

```diff
@@ -22,7 +24,14 @@ def create(
             u"name": name,
             u"subject": subject,
         }
-        return self._connection.post(self._uri_prefix, json=data)
+        try:
+            return self._connection.post(self._uri_prefix, json=data)
+        except Py42BadRequestError as err:
+            if u"NAME_EXISTS" in err.response.text:
+                raise Py42CaseNameExistsError(err, name)
+            if u"DESCRIPTION_TOO_LONG" in err.response.text:
```

Comment:
even though programmatically has no effect, it is still better to use `elif`

Here is why:  Say this code changes and no longer raised the error now, we may forget to change it to `elif` then.  Having it now is more readable as well.

Reply:
Added.